src/matchers/run-query.js

```diff
@@ -45,6 +50,6 @@ async function extract(url, params, log = console) {
 
 module.exports = {
   required: [],
-  pattern: (url) => /.*run_query.*/.test(url),
+  pattern: (url) => /(^https:\/\/adobeioruntime.net\/api\/v1\/web\/helix\/helix-services\/run-query@.*)|(^\/?run-query.*)/.test(url),
```

Comment:
```suggestion
  pattern: (url) => /(^https:\/\/adobeioruntime\.net\/api\/v1\/web\/helix\/helix-services\/run-query@.*)|(^\/?run-query.*)/.test(url),
```

One unescaped `.`

Reply:
For the second part, it might be useful to actually parse the URL with  `^\/?run-query.*).test(new URL(url).pathname)` https://nodejs.org/api/url.html#url_url_pathname

The `url` value is the full URL, including hostname, but you only need to compare the path. (Tests would be good)

Reply:
right ok :) thanks for this pointer